src/openforms/emails/digest.py

```diff
@@ -492,3 +525,165 @@ def _report(var_key: str = "", exception: bool = False, rule_index: int = 0):
             )
 
     return invalid_logic_rules
+
+
+def collect_expired_or_near_expiry_referentielijsten_data():
+    config = GlobalConfiguration.get_solo()
+
+    service_tabels_items_mapping = {}
+    live_forms = Form.objects.live()
+
+    # check only the services that are used by active forms and by specific components
+    for form in live_forms:
+        components = form.iter_components()
+
+        for component in components:
+            if component["type"] in ["selectboxes", "radio", "select"] and (
+                component.get("openForms", {}).get("dataSrc")
+                == DataSrcOptions.referentielijsten
+            ):
+                service = config.referentielijsten_services.filter(
+                    slug=component["openForms"]["service"]
+                ).first()
+
+                if service:
+                    with build_client(
+                        service, client_factory=ReferentielijstenClient
+                    ) as client:
+                        try:
+                            response = client.get(
+                                "tabellen",
+                                params={"code": component["openForms"]["code"]},
+                            )
+                        except RequestException:
+                            response = []
+
+                        service_label = service.label
+                        tabels = list(pagination_helper(client, response.json()))
```

Comment:
```suggestion
                        tables = list(pagination_helper(client, response.json()))
```
please, English in code.